gui/system/models.py

```diff
@@ -1212,5 +1235,58 @@ def is_enabled(self):
         return self.is_available(support=self)[0] and self.enabled
 
 
+class ACMERegistrationBody(models.Model):
+    contact = models.EmailField(
+        verbose_name='Contact Email'
+    )
+    status = models.CharField(
+        verbose_name='Status',
+        max_length=10
+    )
+    key = models.TextField(
+        verbose_name='JWKRSAKey'
+    )
+    acme = models.ForeignKey(
+        'ACMERegistration',
+        on_delete=models.CASCADE,
+    )
+
+
+class ACMERegistration(models.Model):
+    uri = models.URLField(
+        verbose_name='URI'
+    )
+    directory = models.URLField(
+        verbose_name='Directory URI',
+        unique=True  # Is this the right step ?
```

Comment:
Right, why unique? Does every new acme certificate generate one entry here? Or is it only one per provider (Lets encrypt?)